packages/ts/react-crud/src/autogrid-columns.tsx

```diff
@@ -22,7 +22,8 @@ export type ColumnOptions = Omit<GridColumnProps<any>, 'dangerouslySetInnerHTML'
 function getTypeColumnOptions(propertyInfo: PropertyInfo): ColumnOptions {
   // eslint-disable-next-line default-case
   switch (propertyInfo.type) {
-    case 'number':
+    case 'integer':
+    case 'decimal':
```

Comment:
For auto-grid both types render as integers for now, same as before. I didn't want to bloat up the PR any further. Let's address that in a separate PR.